llvm/include/llvm/TargetParser/PPCTargetParser.def

```diff
@@ -0,0 +1,80 @@
+#ifndef PPC_FEATURE
```

Comment:
Missing file description similar to the `head` of `clang/include/clang/Basic/Builtins.def`